### PR TITLE
Remove Final Travis CI Pieces

### PR DESCRIPTION
**Pull request by HighMileage**
## What
Remove references to Travis CI -- this includes the badge for the Travis CI builds and the `.travis.yml` file.

## Why
Builds have been moved over to GitHub actions so this link and file are no longer needed.

@coopergillan

### CODE REVIEW

**Comment by coopergillan** [comment]
@ClashTheBunny

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Runbook
 
 [![Gem Version](https://badge.fury.io/rb/runbook.svg)](https://badge.fury.io/rb/runbook)
-[![Build Status](https://travis-ci.org/braintree/runbook.svg?branch=master)](https://travis-ci.org/braintree/runbook)
+[![Build Status](https://github.com/braintree/runbook/actions/workflows/ruby.yml/badge.svg?branch=master&event=push)](https://github.com/braintree/runbook/actions?query=branch%3Amaster+event%3Apush)
```
Comment (coopergillan) [inline README.md:4]: Nice catch!

**Review by coopergillan — APPROVED** [review]
🚋

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 [![Gitter](https://badges.gitter.im/braintree/runbook.svg)](https://gitter.im/braintree/runbook?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (coopergillan) [inline README.md:5]: Should we ditch this also as long as we are in here? It looks like no action in over a year and Patrick is gone now.

Comment (HighMileage) [reply README.md]: I removed this thinking that GitHub issues can fill the gap

**Review by pblesi — APPROVED** [review]

**Comment by coopergillan** [comment]
Thanks @pblesi 👍

**Pull request merged**